src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), getRandomNumber());
+    }
+
+    public int getRandomNumber() {
```

Comment:
The method `getRandomNumber` can be private since it is only used within the `Lottery` class and is not intended to be exposed as part of the class's public API.